util/bitutil/bitutil.go

```diff
@@ -0,0 +1,6 @@
+package bitutil
+
+// FlagSetInByte returns true iff the `pos` bit is set to true in `flags`.
+func FlagSetInByte(flags byte, flag byte) bool {
```

Comment:
nitpicky: I might rename `flag` to `index` or `offset` or simply `bit` and document it as being position from the right.